src/main/java/com/exactpro/th2/act/ActHandler.java

```diff
@@ -449,7 +449,7 @@ private Message backwardCompatibilityConnectionId(PlaceMessageRequest request) {
         return Message.newBuilder(request.getMessage())
                 .mergeMetadata(MessageMetadata.newBuilder()
                         .mergeId(MessageID.newBuilder()
-                                .setConnectionId(request.getConnectionId())
+                                .setConnectionId(request.getMessage().getMetadata().getId().getConnectionId())
```

Comment:
This change has been done for backward compatibility
The whole backwardCompatibilityConnectionId method can be deleted after removing deprecated fields from gRPC

Reply:
Sure, I already removed it in th2-grpc-act-template, please see https://github.com/th2-net/th2-grpc-act-template/compare/th2-2150-books-pages